maker/src/cfd.rs

```diff
@@ -19,10 +20,13 @@ use model::TxFeeRate;
 use nonempty::NonEmpty;
 use std::collections::HashMap;
 use time::Duration;
+use time::OffsetDateTime;
 use xtra::prelude::MessageChannel;
 use xtra_productivity::xtra_productivity;
 use xtras::SendAsyncSafe;
 
+const ROLLOVER_PARAMS_TTL: Duration = Duration::MINUTE;
```

Comment:
> Wouldn't it be better that this is passed in through the HTTP API?

It would give us more flexibility, but the value seems limited unless we anticipate ever wanting to change it. Do you think it's worth it to add it to the `OfferParams`?

> Also, the funding rate might have a way longer expiry, something like <8h.

Good point. The value I chose was extremely arbitrary.

Reply:
@da-kami, @bonomat: At first I was going to go with your suggestions, but I don't want to rush to make hasty changes to a HTTP API that I think already has some problems. I've opened a discussion about this topic: https://github.com/itchysats/itchysats/discussions/2886.

I will also open a ticket to make this time-to-live configurable.

Finally, I'm gonna set the constant to 5 minutes. It's true that the value should live for 8 hours, but the updates to the offer parameters are so frequent that using that value would lead to problems e.g. setting it to 8 hours a few minutes before the value actually changes and then never hearing back from the API because of a problem, leading to allowing rollovers with stale data for an extended period of time.

2 minutes is also fine, but I don't think we have to cut it so close.